CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,50 @@
+# Phlowers Code of Conduct
+
+
+## Introduction
+
+This Code of Conduct applies to all spaces managed by the NumPy project, including all public and private mailing lists, issue trackers, wikis, blogs, Twitter, and any other communication channel used by our community. The NumPy project does not organise in-person events, however events related to our community should have a code of conduct similar in spirit to this one.
```

Comment:
You cannot make a copy past, it should remain an inspiration. First, you have many reference to the Numpy project. And then, I think you should remain "simple" : we just have a slack and a github (issues, PRs). We are not particpating in events right now.